app/services/contracts/question-state-controller-service.server.ts

```diff
@@ -0,0 +1,27 @@
+import questionStateControllerJson from "contracts/QuestionStateController.json";
+import { BigNumber, ethers } from "ethers";
+import type { ContractContext } from "~/types/generated/QuestionStateController";
+import { questionStateEnum } from "~/utils/helpers";
+
+const GET_QUESTIONS_OFFSET = 1000;
+
+export default class QuestionStateControllerService {
+  private questionStateController: ContractContext;
+
+  constructor(provider: ethers.providers.Provider) {
+    this.questionStateController = new ethers.Contract(
+      questionStateControllerJson.address,
+      questionStateControllerJson.abi,
+      provider
+    ) as unknown as ContractContext;
```

Comment:
this slightly confuses me - is this needed? Or maybe put a TODO to update unknown - not a biggie.